std_msgs/Header header
float64 targetY             # targetY reading
float64 targetY_variance    # targetY_variance reading
